e2e-tests/tests/monitoring/06-check-pgstatstatements-query-source.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      kubectl -n ${NAMESPACE} patch perconapgcluster/${test_name} --type=merge -p '{"spec":{"pmm":{"querySource":"pgstatstatements"}}}'
+
+      sleep 60
+
```

Comment:
can we do better than sleep? if not, please add a descriptive message to explain why is it waiting

Reply:
I don't think we need anything more sophisticated than a simple sleep here, but a comment is definitely required. 
[e70b50b](https://github.com/percona/percona-postgresql-operator/pull/870/commits/e70b50b03e324dc4260e140c22105abbd45736f7)